sdk/test-utils/perfstress/src/utils.ts

```diff
@@ -16,6 +18,14 @@ export function getEnvVar(name: string) {
   return val;
 }
 
+export const httpsAgent = new https.Agent({
+  rejectUnauthorized: false
+  // pfx: require("fs").readFileSync(
+  //   "/workspaces/azure-sdk-for-js/eng/common/testproxy/dotnet-devcert.pfx"
+  // ),
+  // passphrase: "password"
```

Comment:
This is throwing an error. 😕

Reply:
I would omit the PFX option for now.  All the other languages just implement the "--insecure" option to disable cert validation.  I don't think it's necessary, but if we really wanted to support a PFX file and password, these would need to be added as an alternate option to the "--insecure" option.

Reply:
Logged an issue for `.pfx` https://github.com/Azure/azure-sdk-for-js/issues/17961